### ISSUE TITLE
ElasticsearchRestClientFactoryBean : default port should be 9200, not 9300

### ISSUE THREAD

**Issue by MarneusCalgarXP** [dadoonet/spring-elasticsearch#149]
current code in spring-elasticsearch 6.1 : 
(ElasticsearchRestClientFactoryBean )

```java
private Tuple<String, Integer> toAddress(String address) throws UnknownHostException {
		if (address == null) return null;

		String[] splitted = address.split(":");
		int port = 9300;
		if (splitted.length > 1) {
			port = Integer.parseInt(splitted[1]);
		}

		return new Tuple<>(splitted[0], port);
	}
```

a REST HTTP client should bind port 9200 and not 9300

**Comment by dadoonet** [comment]
Oh! That's true. Wanna send a PR?

**Comment by MarneusCalgarXP** [comment]
sorry, I haven't set up my workstation for github now (I'm at work)

**Comment by dadoonet** [comment]
No problem. Note that you can edit files directly from GitHub. This is a small change so if you wish, feel free to contribute the patch.
Otherwise I'll do it.

**Comment by MarneusCalgarXP** [comment]
OK, here's the PR:
* https://github.com/dadoonet/spring-elasticsearch/pull/150

**Comment by dadoonet** [comment]
Closed by #150
